tests/common/test_command_populate_aip_checksums.py

```diff
@@ -0,0 +1,287 @@
+    )
+
+
+@pytest.mark.django_db
+@mock.patch(
+    "archivematica.storage_service.common.utils.get_compressed_package_checksum"
+)
+def test_command_for_compressed_aip(
+    get_compressed_package_checksum: mock.Mock,
+    capsys: pytest.CaptureFixture[str],
+    compressed_package: models.Package,
+    aip_storage_fs_location: models.Location,
+) -> None:
+    compressed_package_checksum = (
+        "c2924159fcbbeadf8d7f3962b43ec1bf301e1b4f12dd28a8b89ec819f3714747"
+    )
+    algo = "sha256"
+    get_compressed_package_checksum.return_value = (
+        compressed_package_checksum,
+        algo,
+    )
+
+    call_command(
+        "populate_aip_checksums",
+        "--location-uuid",
+        aip_storage_fs_location.uuid,
+    )
+
+    captured = capsys.readouterr()
+    assert captured.out.split("\n")[:-1] == [
+        f"AIP {compressed_package.uuid} updated with {algo} checksum {compressed_package_checksum}",
+        "Complete. Checksums added for all 1 identified AIPs.",
+    ]
+
+
+@pytest.mark.django_db
+@mock.patch("archivematica.storage_service.common.utils.generate_checksum")
+def test_command_for_uncompressed_aip(
+    generate_checksum: mock.Mock,
+    capsys: pytest.CaptureFixture[str],
+    uncompressed_package: models.Package,
+    aip_storage_fs_location: models.Location,
+) -> None:
+    uncompressed_package_checksum = "098f6bcd4621d373cade4e832627b4f9"
+    generate_checksum.return_value = mock.Mock(
+        **{
+            "hexdigest.return_value": uncompressed_package_checksum,
+        }
+    )
+
+    call_command(
+        "populate_aip_checksums",
+        "--location-uuid",
+        aip_storage_fs_location.uuid,
+    )
+
+    captured = capsys.readouterr()
+    assert captured.out.split("\n")[:-1] == [
+        f"AIP {uncompressed_package.uuid} updated with sha256 checksum {uncompressed_package_checksum}",
+        "Complete. Checksums added for all 1 identified AIPs.",
+    ]
+
+
+@pytest.mark.django_db
+@mock.patch(
+    "archivematica.storage_service.common.utils.get_compressed_package_checksum"
+)
+@mock.patch("archivematica.storage_service.common.utils.generate_checksum")
+def test_command_to_download_aips(
+    generate_checksum: mock.Mock,
+    get_compressed_package_checksum: mock.Mock,
+    capsys: pytest.CaptureFixture[str],
+    compressed_package: models.Package,
+    uncompressed_package: models.Package,
+    aip_storage_fs_location: models.Location,
+) -> None:
+    uncompressed_package_checksum = "098f6bcd4621d373cade4e832627b4f9"
+    generate_checksum.return_value = mock.Mock(
+        **{
+            "hexdigest.return_value": uncompressed_package_checksum,
+        }
+    )
+    compressed_package_checksum = (
+        "c2924159fcbbeadf8d7f3962b43ec1bf301e1b4f12dd28a8b89ec819f3714747"
+    )
+
+    algo = "sha256"
+    get_compressed_package_checksum.return_value = (
+        compressed_package_checksum,
+        algo,
+    )
+    total_packages = len(models.Package.objects.all())
```

Comment:
Would `models.Package.objects.count()` work better here and the test below? Also you could inline the call in the f-string and remove the `total_packages` variables.